### PR TITLE
Add select component

### PR DESCRIPTION
**Pull request by sandersky**
# PATCH

### CODE REVIEW

File: tests/dummy/app/pods/select/template.hbs [hunk 1]
```diff
@@ -0,0 +1,241 @@
+<div class="section">
```
Comment (vesper2000) [inline tests/dummy/app/pods/select/template.hbs:1]: Update on-change to onChange, there are a couple of places that need to be changed.

Comment (sandersky) [inline tests/dummy/app/pods/select/template.hbs:1]: Thanks for catching this; problem has been addressed.

Comment (vesper2000) [inline tests/dummy/app/pods/select/template.hbs:1]: Also need to add a readme.

File: tests/dummy/app/pods/select/template.hbs [hunk 2]
```diff
@@ -0,0 +1,241 @@
...
+  <div class="title">frost-select</div>
+</div>
+<div class="section">
+  <div class="example">
+    <div class="title">Local filtering</div>
+    <div class="snippet">
+      {{format-markdown "```handlebars
+{{frost-select
+  data=data
+  onChange=(action 'onChange')
```
Comment (vesper2000) [inline tests/dummy/app/pods/select/template.hbs]: We are also changing all actions to have consistent name.
So `onChange=(action 'onChange')` would become `onChange=(action 'onChangeHandler')`.

File: tests/dummy/app/pods/select/template.hbs (hunk 1 above)
Comment (sandersky) [inline tests/dummy/app/pods/select/template.hbs:1]: Just did that as well.

File: tests/dummy/app/pods/select/controller.js [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+  }),
+
+  selectedIndex: 1,
+  selectedIndices: [1, 2],
+  preSelectedValue: 'Arthur Curry',
+  selectedValues: ['Arthur Curry', 'Adam Meadows'],
+
+  actions: {
+    onChange (values) {
+      console.log('User selected: ' + values)
```
Comment (sglanzer-fire) [inline tests/dummy/app/pods/select/controller.js]: Can you move these to notifications?

File: frost-select.md [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+    "value": "buzz"
+  }
+]
+
+singleSelected = 1
+multipleSelected = [1, 2]
+```
+
+### Simple single select
+```
```
Comment (vesper2000) [inline frost-select.md]: Add handlebars  `"`handlebars" - it will provide the formatted text that will be consistent with other readmes.

**Comment by sglanzer-fire** [comment]
Approved

[Approved with PullApprove](https://pullapprove.com/ciena-frost/ember-frost-core/pull-request/28/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=ciena-frost/ember-frost-core)

**Pull request merged**